src/main/kotlin/com/bq/poeditor/gradle/utils/DateJsonAdapter.kt

```diff
@@ -36,7 +37,11 @@ class DateJsonAdapter : JsonAdapter<Date>() {
     @Throws(IOException::class)
     override fun fromJson(reader: JsonReader): Date {
         val string = reader.nextString()
-        return sdf.parse(string)
+        return try {
+            sdf.parse(string)
+        } catch (e: ParseException) {
+            Date(0)
+        }
```

Comment:
I'd rather check if the API returns an empty string and return `null` in that case, that way we don't return information that the API doesn't provide instead of adding a default value. You'd need to do something like this:
```suggestion
        if (string == null || string.isBlank()) return null
        return sdf.parse(string)
```
You would also need to change the return type of this function from `Date` to `Date?`

Reply:
Hey @adriangl . I'm gonna create a new pr and take it over, since it's also affecting us and the author seems non-responsive. Regarding your comment:
> Also add the change to the Fixed block of the CHANGELOG's Unreleased section so we can keep track of it for the next release.

What would be the exact change that would be in the `CHANGELOG` file?

Reply:
Sure, go ahead! And thanks a lot 😁